pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -205,6 +208,32 @@ func (db *DB) Migration() *migrate.Migration {
 					`CREATE INDEX IF NOT EXISTS idx_bwa_serial ON bandwidth_agreements (serial_num)`,
 				},
 			},
+			{
+				Description: "delete pieces that are stored on drive using 'ttl' old db",
+				Version:     5,
+				Action: migrate.Func(func(log *zap.Logger, mdb migrate.DB, tx *sql.Tx) error {
+					path := db.dbPath
+					if path != "" {
+						path = filepath.Dir(path)
+						files, err := ioutil.ReadDir(path)
```

Comment:
This needs a separate test that it doesn't delete more than needed.

Reply:
added the test case.  done.